src/components/Card.js

```diff
@@ -16,14 +34,32 @@ export default class Card {
     this._cardElement = null;
   }
 
+  isLiked() {
+    if (!this._cardIsLiked) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  _updateIsLiked(cardID) {
+    if (this.isLiked(cardID)) {
+      this._cardIsLiked = "true";
+      this._handleLikeButton();
+    } else {
+      this._cardIsLiked = "false";
+      this._handleLikeButton();
+    }
+  }
+
   //=================Event Listeners==========================\\
   _setEvenetListeners() {
     this._likeButton.addEventListener("click", () => {
-      this._handleLikeButton();
+      this._updateIsLiked(this._cardID);
```

Comment:
When i click on the like button only like state changes, but here it's needed to update likes on the server